doc/source/contribution/documentation/introduction.md

```diff
@@ -155,6 +155,36 @@ For example, to preview the Japanese translation of the {doc}`introduction` page
 % open ../groonga.doc/doc/ja/html/contribution/documentation/introduction.html
 ```
 
+## Submit contributions
+
+You can submit your contributions to the Groonga repository on GitHub via a pull request.
+Feel free to send a pull request by following two steps.
+
+- Prepare your pull request
+- Submit your pull request
+
+### Prepare your pull request
+
+Ensure your changes are committed and then push your changes to your fork repository on GitHub.
+Follow these commands:
+
+```console
+% git add doc/source/\*.{md,rst}
+# Optional: add translate documentations when you translate them.
+% git add doc/locale/ja/LC_MESSAGES/*.po
+% git commit -m 'Describe your works here'
+% git push
+```
```

Comment:
Is it a push to the `main` branch?
I think a working branch would be better.

Reply:
fix: [54ca7a9](https://github.com/groonga/groonga/pull/1753/commits/54ca7a9749e67bfdccfdfb8692596d4054e22f23) 
Thanks. At first I thought creating a working branch was late for this step because I usually create the branch before working.
So I wanted to add it at the beginning of this document but I rethought this and decided that adding this step is technically fine.